invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/label.overrides

```diff
@@ -1,6 +1,7 @@
 .ui.label.access-status {
   color: @white;
   width: max-content;
+  z-index: 100;
```

Comment:
I have checked the issue and the underlying problem is not the ui label elemnt but data-tooltip.
an example of why this is not optimal solution is: If you set the z-index of the label and you have a sticky header on top of the page, you will end up with labels "hovering" over the header (or other sticky elements)
I would suggest to go with

```
[data-tooltip]:after {
   z-index:3
}
```